### PR TITLE
Fixes for Cyclops Matrix and moving RSFT to the default position

### PR DESCRIPTION
**Pull request by egla89**
## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* The matrix was off and is now correct.
* RSFT in now at the default location

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by noroadsleft** [comment]
@westfoxtrot ?

**Comment by egla89** [comment]
> The two files in `.vscode/ipch/` should be removed from the pull request.

Done

File: keyboards/westfoxtrot/cyclops/cyclops.h [hunk 1]
```diff
@@ -35,9 +35,9 @@
 ) { \
 	{ K00,  K01,   K02,  K03,   K04,   K05,   K06,  K07,   K08,   K09,   K0A,  K0B,  K0C,  K0D,  K0E }, \
 	{ K10,  KC_NO, K12,  K13,   K14,   K15,   K16,  K17,   K18,   K19,   K1A,  K1B,  K1C,  K1D,  K1E }, \
-	{ K20,  KC_NO, K22,  K23,   K24,   K25,   K26,  K27,   K28,   K29,   K2A,  K2B,  K2C,  K2D,  K2E }, \
+	{ K20,  KC_NO, K22,  K23,   K24,   K25,   K26,  K27,   K28,   K29,   K2A,  K2B,  K2C,  K2D,  K4E }, \
```
Comment (mechmerlin) [inline keyboards/westfoxtrot/cyclops/cyclops.h]: You are editing the electrical matrix here. That position that you changed can ONLY either be K2E or KC_NO.

File: keyboards/westfoxtrot/cyclops/cyclops.h [hunk 2]
```diff
@@ -35,9 +35,9 @@
...
 	{ K30,  K31,   K32,  K33,   K34,   K35,   K36,  K37,   K38,   K39,   K3A,  K3B,  K3C,  K3D,  K3E }, \
-	{ K40,  K41,   K42,  KC_NO, KC_NO, KC_NO, K46,  KC_NO, KC_NO, KC_NO, K4A,  K4B,  K4C,  K4D,  K4E }  \
+	{ K40,  K41,   K42,  KC_NO, KC_NO, KC_NO, K46,  KC_NO, KC_NO, KC_NO, K4A,  K4B,  K4C,  K2E,  K4D }  \
```
Comment (mechmerlin) [inline keyboards/westfoxtrot/cyclops/cyclops.h]: You are editing the electrical matrix here too. Those positions can only either be K4D or K4E or corresponding KC_NO. Please revert this change.
